docs/journalist.rst

```diff
@@ -172,39 +182,82 @@ generated and you can log back in and send a reply.
 Moving Documents to the *Secure Viewing Station*
 ------------------------------------------------
 
-Documents sent by sources can only be viewed on the *Secure Viewing
-Station*. After clicking on an individual source, you will see the
-page below with any messages that source has sent you. Click on a
-document or message name to save it, or select a number of documents
+Step 1: Download the encrypted submission
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Documents and messages sent by sources can only be decrypted and viewed on the
+*Secure Viewing Station*. After clicking on an individual source, you will see the
+page below with any documents or messages that source has sent you. Documents
+always end with ``-doc.gz.gpg``, while messages always end with ``-msg.gpg``.
+
+Click on a document or message name to save it, or select a number of documents
 and save them all at once by clicking **Download Selected**.
 
 |Load external content|
 
-A dialog box will appear asking if you want to **Open** or **Save**
-the file. Select **Save File**:
+A dialog box with two choices will appear, **Cancel** and **Save file**:
 
 |Download selected|
 
-In order to protect you from malware, the browser in Tails will only
-allow you to download documents to a special persistent folder located
-at ``/home/amnesia/Tor Browser``.
+Click **Save file**. In the save dialog, select one
+of the two folders highlighted in red in the screenshot below:
 
 |Download to sandbox folder|
 
-.. tip:: The special folder mentioned here is called **Tor Browser**,
-         not "Persistent." Attempting to download directly into the
-         **Persistent** folder will only lead to frustration.
+The difference between these two folders is as follows:
+
+- **Tor Browser**. Downloads saved to this folder will be stored in memory,
```

Comment:
As far as I can tell, Tails uses a union-mounted [tmpfs](https://en.wikipedia.org/wiki/Tmpfs) filesystem.

```
tmpfs on /lib/live/mount/overlay type tmpfs (rw,noatime,mode=755)
```

Since it uses no swap, this should be equivalent to a RAM disk. That's consistent with the Tails docs:

> The only storage space used by Tails is in RAM, which is automatically erased when the computer shuts down.

https://tails.boum.org/about/index.en.html

Does that make sense? If that's correct, then I think being explicit about it being in-memory storage, it would be my preference to say so explicitly so users don't worry about wiping files that are stored in the non-persistent portion of the filesystem.

Reply:
That is correct, makes sense, thanks for the clarification